phpBB/phpbb/auth/provider/oauth/oauth.php

```diff
@@ -126,8 +133,9 @@ class oauth extends \phpbb\auth\provider\base
 	* @param	\Symfony\Component\DependencyInjection\ContainerInterface $phpbb_container DI container
 	* @param	string			$phpbb_root_path
 	* @param	string			$php_ext
+	* @param 	\phpbb\event\dispatcher_interface	$dispatcher phpBB Event dispatcher object
```

Comment:
Hard to track down but there is actually a space after `@param which the sami parse check does not like. If you remove it, the tests will run fine.